book/controller.rst

```diff
@@ -619,8 +619,7 @@ session.
 Flash Messages
 ~~~~~~~~~~~~~~
 
-You can also store small messages that will be stored on the user's session
-for exactly one additional request. This is useful when processing a form:
+You can also store small messages that will be stored on the user's session. This is useful when processing a form:
```

Comment:
Ok. But what do you think about removing or modify "for exactly one additional request" sentence?. Do you think adding the tip about get()/peek() will be enough without modifiying this sentence?.